Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM rust:latest as build
+WORKDIR /app
+
+RUN apt-get update && apt-get install -y clang protobuf-compiler
+COPY . .
+RUN cargo build
+
```

Comment:
After build, do another `FROM` image and copy the binary over to run. Pick something lighter, such as alpine for that purpose.

Reply:
Will create another PR to do that!